### ISSUE TITLE
No preview for private images

### ISSUE THREAD

**Issue by slaFFik** [imath/buddydrive#66]
A bit unexpected.

![image](https://cloud.githubusercontent.com/assets/282234/14992308/2cea5568-116e-11e6-92a6-7d66a7026b44.png)

**Comment by imath** [comment]
As the file is private, some users asked the thumbnail to be deleted/not created. Some users are using BuddyDrive in private communities. A filter would be nice i guess to let people decide.

**Comment by geonom** [comment]
For public groups I'd imagine thumbnail creation and display to be true by default (with an option to turn it off) - for private groups indeed would be great to add an option, whether they'd want to use thumbnails.

Though if an option per group were difficult to implement, a sidewide option might be sufficient (at least in my case it would be alright).

**Comment by imath** [comment]
yes i guess the option is an interesting road too. I'll probably do it in a future release.
